### DISCUSSION TITLE
[2023-03-20] fsic-5th-meeting

### DISCUSSION THREAD

**Discussion by kevinjantw** [bol-edu/caravel-soc#15] [General]
[fsic-5th-meeting 3-20-2023.pdf](https://github.com/bol-edu/caravel-soc/files/11024720/fsic-5th-meeting.3-20-2023.pdf)

**Comment by wei851246721** [comment]
hi，我想知道该如何连接到pdf中Online FPGA的环境？

**Reply by boledulab** [reply]
需實名認證 請填寫申請表單後會再審核
https://docs.google.com/forms/d/1hfK0vA_avZ0H4KnYhFmH69pmo52hHpTpMq58AdvnQfQ
